### DISCUSSION TITLE
Global midi channel

### DISCUSSION THREAD

**Discussion by Anaximander78** [shanteacontrols/OpenDeck#116] [General]
Hey guys, i just bought an opendeck. What i did not realise is that you cant change the global midi channel, is this correct? So every fader, poti etc... can have its own midi channel, but not the global channel wIth all parameters in it x 16.

So there is the Presets (10, right??) and thats possible, but thats it for now i guess.

Sorry for that basic question, but i could not find the answer in the wiki.

Greetings from Berlin

**Comment by paradajz** [comment]
I'm not sure what you're referring to as "global channel" or "all parameters in it x 16". Each component has configurable channel.

**Reply by Anaximander78** [reply]
Yeah, thanks for answering. I mean it in the way of having 16 different setups with every midi channel. I am working with a zynthian(synth with open source code)  which can run 16 different synthesizers in 16 "Layers". Each "Layer" is a own midi channel instead of a "Preset".  So if i want to play different synthesizers on zynthian with my midikeyboard i just change the midi out channel of my keyboard. 

So my confusion was this, that i thought opendeck organizes the same way.  

But having 10 Presets is totally fine! I will just configure every component in 10 channels accordingly. So Preset 1 will have all components on channel 1 etc... . 

Sorry for the Confusion

**Comment by paradajz** [comment]
FYI this is now implemented on the master branch, just not yet released.

**Comment by paradajz** [comment]
Released: https://github.com/shanteacontrols/OpenDeck/releases/tag/v7.0.0
